src/pypi_attestations/_impl.py

```diff
@@ -191,6 +191,31 @@ def sign(cls, signer: Signer, dist: Distribution) -> Attestation:
         except ConversionError as e:
             raise AttestationError(str(e))
 
+    @property
+    def claims(self) -> dict[x509.ObjectIdentifier, str]:
```

Comment:
Let's stringify these OIDs so that we don't have to return a foreign type here:

```suggestion
    def claims(self) -> dict[x509.ObjectIdentifier, str]:
```